pkg/yurtctl/util/kubernetes/util.go

```diff
@@ -56,7 +56,7 @@ var (
 	// PropagationPolicy defines the propagation policy used when deleting a resource
 	PropagationPolicy = metav1.DeletePropagationForeground
 	// WaitServantJobTimeout specifies the timeout value of waiting for the ServantJob to be succeeded
-	WaitServantJobTimeout = time.Minute * 2
+	WaitServantJobTimeout = time.Minute * 5
```

Comment:
Why do you increase the timeout?

Reply:
In my test env, 2min is too short for the job finish , however, after timeout, the convert job succeeded

Reply:
sure, thanks